src/Microsoft.Sbom.Common/Constants.cs

```diff
@@ -20,4 +20,7 @@ public static class Constants
 
     public const string DefaultManifestInfoName = "SPDX";
     public const string DefaultManifestInfoVersion = "2.2";
+
```

Comment:
I'm guessing that this extra newline is unintended?
```suggestion
```

Reply:
I added it to separate the sections, following the format already in place